src/System.Linq.Parallel/tests/Combinatorial/ParallelQueryCombinationTests.cs

```diff
@@ -0,0 +1,1523 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Linq;
+using Xunit;
+
+namespace Test
+{
+    [Trait("category", "outerloop")]
+    public partial class ParallelQueryCombinationTests
+    {
+        [Theory]
+        [MemberData("UnaryOperators")]
+        [MemberData("BinaryOperators")]
+        public static void Aggregate(LabelledOperation source, LabelledOperation operation)
+        {
+            Assert.Equal(Functions.SumRange(defaultStart, defaultSize),
+                operation.Item(defaultStart, defaultSize, source.Item).Aggregate((x, y) => x + y));
+        }
+
+        [Theory]
+        [MemberData("UnaryOperators")]
+        [MemberData("BinaryOperators")]
+        public static void Aggregate_Seed(LabelledOperation source, LabelledOperation operation)
+        {
+            Assert.Equal(Functions.SumRange(defaultStart, defaultSize),
+                operation.Item(defaultStart, defaultSize, source.Item).Aggregate(0, (x, y) => x + y));
+        }
+
+        [Theory]
+        [MemberData("UnaryOperators")]
+        [MemberData("BinaryOperators")]
+        public static void Aggregate_Result(LabelledOperation source, LabelledOperation operation)
+        {
+            Assert.Equal(Functions.SumRange(defaultStart, defaultSize) + 17,
```

Comment:
What's the relevance of "17" here?  Is it just an arbitrary number?  I'd put it in a named constant to better describe it, and because it's used multiple times and presumably has to match each time.